.rubocop.yml

```diff
@@ -46,7 +46,7 @@ Metrics/AbcSize:
   Max: 391
 
 Metrics/ClassLength:
-  Max: 235
```

Comment:
well... now that I'm thinking about it. This kind of makes the necessity of a future refactor less visible. It might actually be a good thing that the dev has to notice this whenever they touch the class currently.